doc/about/tech.rst

```diff
@@ -15,6 +15,13 @@ L'écosystème Javascript étant à peine naissant et les frameworks allant et v
 
 Ne restait plus que le Python et le Ruby avec les frameworks Django et Ruby On Rails. Ruby ayant une réputation d'être très "cutting edge", c'est Python, un langage bien implanté et ayant fait ses preuves, qui a été retenu.
 
+Il est à noter que réécrire le site avec un framework PHP comme Laravel ou Symphony
```

Comment:
Ça, ça viens d'une spéculation que j'ai, à voir si Skia est d'accord là dessus, c'est lui qui a pris la décision à l'époque

Reply:
Indeed, c'est une spéculation: c'est surtout que 1) je savais pas faire de Ruby, donc ça laissait que PHP ou Python, mais à l'époque, PHP 7 pointait à peine le bout de son nez, et traînait un [sacré paquet de casseroles](https://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/), et 2) Django m'avait été chaudement recommandé par quelques personnes de mon entourage, bien plus expérimentées que moi sur le sujet.  
Notez que ce paragraphe ajouté reste vrai, par contre il a pas été considéré à l'époque.

Reply:
J'ajoute que pour Ruby, je voulais pas trop m'y frotter parce qu'à l'époque, j'avais été en contact avec en essayant de déployer un Gitlab (pas pour l'AE, ailleurs), mais y'avait pas encore le paquet Omnibus, donc fallait le déployer un peu comme un développeur, en gérant tout Ruby soit même, et j'avais eu tellement de problèmes avec la gestion des dépendances pour avoir chaque paquet dans la bonne version, que je voulais vraiment pas m'y frotter. À côté de ça, y'avait Python où tu faisais un `venv` et `pip install`, ça marchait, c'était nickel.

Reply:
ok ok, donc ça va, j'ai pas sorti de conneries